.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - next
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
+      - name: Build library
+        run: npm run build
+      - name: Run tests
+        run: npm run test
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does it need the GITHUB token?

Reply:
So that it can generate a [Release](https://github.com/truework/truework-sdk-node/releases) and push git tags back to the repo. Our current releases were generated by `np`.

I'm not actually positive that we'll be able to do this without either using a personal auth token (not ideal) or unprotecting the master branch. This dry run should tell us if it has access to Github or not.